internal/auth0/prompt/resource_custom_text.go

```diff
@@ -17,10 +17,11 @@ import (
 
 var (
 	availablePrompts = []string{
-		"common", "consent", "device-flow", "email-otp-challenge", "email-verification", "invitation", "login",
-		"login-id", "login-password", "login-passwordless", "login-email-verification", "logout", "mfa", "mfa-email",
-		"mfa-otp", "mfa-phone", "mfa-push", "mfa-recovery-code", "mfa-sms", "mfa-voice", "mfa-webauthn",
-		"organizations", "reset-password", "signup", "signup-id", "signup-password", "status",
+		"capcha", "common", "consent", "custom-form", "customized-consent", "device-flow", "email-otp-challenge",
+		"email-verification", "invitation", "login", "login-id", "login-password", "login-passwordless",
+		"login-email-verification", "logout", "mfa", "mfa-email", "mfa-otp", "mfa-phone", "mfa-push", "mfa-recovery-code",
+		"mfa-sms", "mfa-voice", "mfa-webauthn", "organizations", "passkeys", "phone-identifier-challenge",
+		"phone-identifier-enrollment", "reset-password", "signup", "signup-id", "signup-password", "status",
```

Comment:
```suggestion
		"captcha", "common", "consent", "custom-form", "customized-consent", "device-flow", "email-otp-challenge",
		"email-verification", "invitation", "login", "login-email-verification", "login-id", "login-password",
		"login-passwordless", "logout", "mfa", "mfa-email", "mfa-otp", "mfa-phone", "mfa-push", "mfa-recovery-code",
		"mfa-sms", "mfa-voice", "mfa-webauthn", "organizations", "passkeys", "phone-identifier-challenge",
		"phone-identifier-enrollment", "reset-password", "signup", "signup-id", "signup-password", "status"
```

Reply:
It seems to be invalid without a trailing comma, so I'll added it
```
terraform-provider-auth0 on  add-new-prompt-values-to-auth0_prompt_custom_text via 🐹 v1.22.1 on ☁️   
❯ make docs
 ::  Generating docs                                                              
rendering website for provider "terraform-provider-auth0" (as "terraform-provider-auth0")
copying any existing content to tmp dir
exporting schema from Terraform
compiling provider "auth0"
2024/07/12 18:45:21 error executing "/usr/local/Cellar/go/1.22.1/libexec/bin/go", [go build -o /var/folders/2t/kxlpb5797yx1zccty0bg42vwb_bpfs/T/tfws2095941905/plugins/registry.terraform.io/hashicorp/auth0/0.0.1/darwin_amd64/terraform-provider-auth0]
2024/07/12 18:45:21 # github.com/auth0/terraform-provider-auth0/internal/auth0/prompt
internal/auth0/prompt/resource_custom_text.go:24:102: syntax error: unexpected newline in composite literal; possibly missing comma or }
Error executing command: unable to generate website: error exporting provider schema from Terraform: unable to execute go build command: error executing "/usr/local/Cellar/go/1.22.1/libexec/bin/go": exit status 1

exit status 1
main.go:14: running "go": exit status 1
make: *** [docs] Error 1
```

Reply:
Sure!